conf/logrotate.conf

```diff
@@ -20,6 +20,23 @@ notifempty
   endscript
 }
 
+## AUTH
+/var/log/st2/st2auth.log {
```

Comment:
We only run one instance of st2auth process and don't include PID in the log file name (like in the action runner case) so I believe this is fine.